acorn-to-esprima.js

```diff
@@ -187,10 +187,6 @@ var astTransformVisitor = {
       return node.expression;
     }
 
-    if (t.isFlow(node)) {
```

Comment:
So then we need to remove this line? https://github.com/babel/babel-eslint/pull/89#issuecomment-101579831